### PR TITLE
New use case sample code for clickstream analytics dashboard

### PR DESCRIPTION
**Pull request by Boburmirzo**
Added a code sample for demonstrating a clickstream analytics dashboard using Google Analytics Data API, Streamlit and Plotly.

### CODE REVIEW

**Comment by ashish-bagri** [comment]
perhaps the dashboard.py can be renamed to consumer.py to have naming consistency across usecases

File: use-cases/clickstream-analytics-dashboard/dashboard.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+st.title("Clickstream Analytics Dashboard")
+
+placeholder = st.empty()
+
+
+def update_dashboard(data):
+    if data:
+        df = pd.DataFrame(data)
+
```
Comment (ashish-bagri) [inline use-cases/clickstream-analytics-dashboard/dashboard.py:28]: is the "data" a single row or not? why a dataframe for a single row?

Comment (ashish-bagri) [reply use-cases/clickstream-analytics-dashboard/dashboard.py:28]: Ok, i see that "data" is a global variable of type list which is appended with new data and the dashboard is refreshed on every update.

**Comment by Boburmirzo** [comment]
> perhaps the dashboard.py can be renamed to consumer.py to have naming consistency across usecases

Yes, I will rename it to consumer.

File: use-cases/clickstream-analytics-dashboard/dashboard.py (hunk 1 above)
Comment (Boburmirzo) [reply use-cases/clickstream-analytics-dashboard/dashboard.py:28]: Yes, correct data rows are increased to keep previous events and update the dashboard with all events. I tried to simulate somehow stateful tables update:)

**Review by PabloPardoGarcia — COMMENTED** [review]
Looks and works great! 👍

I've ran it with the `fake_producer.py`, did you create the GA setup on our GCP account?

**Comment by Boburmirzo** [comment]
> Looks and works great! 👍
> 
> I've ran it with the `fake_producer.py`, did you create the GA setup on our GCP account?

Thanks! Yes, I used my GlassFlow Google account to test it out the docs analytics streaming. There is an instruction in the Readme file on how to create a new Service Account for GA:) 

You will need a JSON key file placed in the same folder and properly id for GA.

**Comment by Boburmirzo** [comment]
I am merging this PR then, start to create a tutorial. Thanks!

**Pull request merged**